models/lido/liquidity/linea/lido_liquidity_linea_syncswap_pools.sql

```diff
@@ -0,0 +1,291 @@
+
+ {% if not is_incremental() %}
+ WHERE DATE_TRUNC('day', s.evt_block_time) >= DATE '{{ project_start_date }}'
+ {% else %}
+ WHERE DATE_TRUNC('day', s.evt_block_time) >= DATE_TRUNC('day', NOW() - INTERVAL '1' day)
+ {% endif %}
+
+ and s.contract_address in (select address from pools)
+group by 1,2
+)
+
+, trading_volume_hourly AS (
+    SELECT
+      s.time,
+      pool,
+      wsteth_amount,
+      p.price,
+      COALESCE((p.price * wsteth_amount) / CAST(POWER(10, 18) AS DOUBLE), 0) AS volume
+    FROM
+      swap_events_hourly AS s
+      LEFT JOIN wsteth_prices_hourly_with_lead AS p ON s.time >= p.time
+      AND s.time < p.next_time
+  )
+  
+, trading_volume AS (
+    SELECT DISTINCT
+      DATE_TRUNC('day', time) AS time,
+      pool,
+      SUM(volume) AS volume
+    FROM trading_volume_hourly
+    GROUP BY 1, 2
+    )
+  
+, all_metrics AS (
+    SELECT
+      l.pool,
+      pools.blockchain,
+      pools.project,
+      0.0 as fee,
+      '' as pool_type,
+      cast(l.time as date) as time,
+      CASE WHEN l.token0 = 0xB5beDd42000b71FddE22D3eE8a79Bd49A568fC8F THEN l.token0 ELSE l.token1 END AS main_token,
+      CASE WHEN l.token0 = 0xB5beDd42000b71FddE22D3eE8a79Bd49A568fC8F THEN p0.symbol ELSE p1.symbol END AS main_token_symbol,
+      CASE WHEN l.token0 = 0xB5beDd42000b71FddE22D3eE8a79Bd49A568fC8F THEN l.token1 ELSE l.token0 END AS paired_token,
+      CASE WHEN l.token0 = 0xB5beDd42000b71FddE22D3eE8a79Bd49A568fC8F THEN p1.symbol ELSE p0.symbol END AS paired_token_symbol,
+      CASE WHEN l.token0 = 0xB5beDd42000b71FddE22D3eE8a79Bd49A568fC8F THEN  amount0/CAST(POWER(10,p0.decimals) as double) ELSE amount1/CAST(POWER(10,p1.decimals) as double) END AS main_token_reserve,
+      CASE WHEN l.token0 = 0xB5beDd42000b71FddE22D3eE8a79Bd49A568fC8F THEN amount1/CAST(POWER(10, p1.decimals) as double) ELSE amount0/CAST(POWER(10, p0.decimals) as double) END AS paired_token_reserve,
+      CASE WHEN l.token0 = 0xB5beDd42000b71FddE22D3eE8a79Bd49A568fC8F THEN p0.price ELSE p1.price END AS main_token_usd_price,
+      CASE WHEN l.token0 = 0xB5beDd42000b71FddE22D3eE8a79Bd49A568fC8F THEN p1.price ELSE p0.price END AS paired_token_usd_price,
+      coalesce(volume,0) AS trading_volume
+    FROM
+      pool_liquidity AS l
+      LEFT JOIN pools ON l.pool = pools.address
+      LEFT JOIN tokens AS t0 ON l.token0 = t0.token
+      LEFT JOIN tokens AS t1 ON l.token1 = t1.token
+      LEFT JOIN tokens_prices_daily AS p0 ON l.time = p0.time
+      AND l.token0 = p0.token
+      LEFT JOIN tokens_prices_daily AS p1 ON l.time = p1.time
+      AND l.token1 = p1.token
+      LEFT JOIN trading_volume AS tv ON l.time = tv.time
+      AND l.pool = tv.pool
+ )
+ 
+
+select  CONCAT(CONCAT(CONCAT(CONCAT(CONCAT(blockchain,CONCAT(' ', project)) ,' '), coalesce(paired_token_symbol,'unknown')),':') , main_token_symbol, ' ', pool_type, ' ', format('%,.3f',round(coalesce(fee,0),4))) as pool_name,
```

Comment:
This line may be too complicated, we may simplify string concat using `||`:
```suggestion
SELECT 
    blockchain || ' ' || project || ' ' || COALESCE(paired_token_symbol, 'unknown') || ':' || main_token_symbol || ' ' || pool_type || ' ' || FORMAT('%,.3f', ROUND(COALESCE(fee, 0), 4)) AS pool_name,
```